config/database/init.go

```diff
@@ -0,0 +1,26 @@
+package database
+
+import (
+	"os"
+
```

Comment:
Please remove the space

Reply:
You mean the new line? It's there just to separate between system's libs (Go built-in libraries) and external libs (e.g., from github).